kubernetes_topology/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "maintainer": "info@stackstate.com",
+  "manifest_version": "0.1.0",
+  "max_agent_version": "6.0.0",
+  "min_agent_version": "5.12",
```

Comment:
for the min and max versions, what is checked here; their version or our version number?

Reply:
Our version, this is only relevant when building checks separately from the main agent